plugins/woocommerce/includes/emails/class-wc-email.php

```diff
@@ -578,8 +577,8 @@ public function get_cc_recipient() {
 		 * @param object   $object The object (ie, product or order) this email relates to, if any.
 		 * @param WC_Email $email  WC_Email instance managing the email.
 		 */
-		$cc  = apply_filters( 'woocommerce_email_cc_recipient_' . $this->id, $cc, $this->object, $this );
-		$ccs = array_map( 'trim', explode( ',', $cc ) );
+		$cc  = apply_filters( 'woocommerce_email_cc_recipient_' . $this->id, $this->cc, $this->object, $this );
+		$ccs = array_map( 'trim', explode( ',', $cc ?? '' ) );
```

Comment:
Would it be worth applying the same approach for [get_recipient()](https://github.com/woocommerce/woocommerce/blob/d3e65e14bb6341ac69fbdd99bda7d9f69e975641/plugins/woocommerce/includes/emails/class-wc-email.php#L561)?

Reply:
Thanks for the review, @yuliyan. I applied it in https://github.com/woocommerce/woocommerce/pull/58037/commits/3cd5c8440136be9bc62a7a1cff88b0a7cc5a7fcb. While the recipients should always be set to something (admin or customer email), technically the filter can change that, and there is no harm in adding the fix anyway.

Since this is CFE, I'm still waiting for https://github.com/woocommerce/woocommerce/issues/58043 to be approved before merging to the release branch.